nixos/modules/system/boot/networkd.nix

```diff
@@ -146,12 +146,13 @@ let
   # .network files have a [Link] section with different options than in .netlink files
   checkNetworkLink = checkUnitConfig "Link" [
     (assertOnlyFields [
-      "MACAddress" "MTUBytes" "ARP" "Unmanaged"
+      "MACAddress" "MTUBytes" "ARP" "Unmanaged" "RequiredForOnline"
     ])
     (assertMacAddress "MACAddress")
     (assertByteFormat "MTUBytes")
     (assertValueOneOf "ARP" boolValues)
     (assertValueOneOf "Unmanaged" boolValues)
+    (assertValueOneOf "RquiredForOnline" boolValues)
```

Comment:
@Mic92 @bgamari I’m sure this should’ve been `RequiredForOnline`, with an _e_. (Given that total my experience with Nix is about six hours, I don’t know why or whether it builds with the typo. Sorry for not writing a patch.)